spec/requests/chatops/slack_slash_command_list_controller_spec.rb

```diff
@@ -0,0 +1,105 @@
+require "rails_helper"
+
+RSpec.describe "SlackSlashCommandListController", type: :request do
+  scenario "responds with default message when no channels configured" do
+    allow(Matchmaking).to receive(:config).and_return({})
+
+    signed = signed_request_body(params: {
```

Comment:
Proposing this as a simple helper to manage the params in tests (see private method in this file below). `signed_request_body` returns an object that gives us the params (body) as a string instead of a hash, a signature, and the timestamp used to produce the signature. These can then be used to make the test request.

If this patterns works then we can extract this into a test helper method and update the other tests.